exporter/signalfxexporter/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/splunk v0.36.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/batchperresourceattr v0.36.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/experimentalmetricmetadata v0.36.0
-	github.com/shirou/gopsutil v3.21.8+incompatible
+	github.com/shirou/gopsutil/v3 v3.21.8
```

Comment:
Could you bump this to v3.21.9? (It fixes a bug that I am trying to get rid off in #5544

Reply:
@mx-psi 
Sure, please note:
Before bumping signalfxexporter to gopsutil/v3 v3.21.8, `make test` failed locally with 1 duplicate symbol error as tracked in #5425 
However, after bumping signalfxexporter gopsutil/v3 v3.21.8, running `make test` fails locally with 4 duplicate symbols(instead of 1 duplicate symbol prior):
```
/usr/local/go/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
duplicate symbol '_get_temperature' in:
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000024.o
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000033.o
duplicate symbol '_open_smc' in:
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000024.o
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000033.o
duplicate symbol '_close_smc' in:
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000024.o
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000033.o
duplicate symbol '_readdrivestat' in:
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000036.o
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3864802333/000039.o
ld: 4 duplicate symbols for architecture x86_64
```
I believe the other 3 duplicate symbol errors need to be resolved(in gopsutil library) as well, please let me know how to proceed. 
Here is the latest output from `make test` after bumping to v3.21.9:
```
/usr/local/go/pkg/tool/darwin_amd64/link: running clang failed: exit status 1
duplicate symbol '_get_temperature' in:
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3198491705/000024.o
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3198491705/000033.o
duplicate symbol '_open_smc' in:
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3198491705/000024.o
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3198491705/000033.o
duplicate symbol '_close_smc' in:
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3198491705/000024.o
    /var/folders/6y/27qzvqc137v5dsm_m0f39_900000gs/T/go-link-3198491705/000033.o
ld: 3 duplicate symbols for architecture x86_64
```

Reply:
That's funny :) I guess I will have to open another PR to fix these new duplicate symbol errors upstream. It can be dealt with separately from this